Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -322,6 +375,40 @@ namespace gdjs {
       this._downKey = true;
     }
   }
+
+  export interface BasisTransformation {
+    toScreen(worldPoint: FloatPoint, screenPoint: FloatPoint): void;
+  }
+
+  export class IsometryTransformation implements BasisTransformation {
+    screen: float[][];
+
+    constructor(angle: float) {
+      const alpha = Math.asin(Math.tan(angle));
```

Comment:
Should the angle be checked/clamped so that it does not go out of some reasonable bounds?

Reply:
(there is the check in the editor side, but maybe we should mirror it here? Wdyt?)

Reply:
Yes, of course. I also took one degree margin to be sure not to fall in limit cases.